notebooks/ZipCodes_Stops_PiP_cuSpatial.ipynb

```diff
@@ -135,33 +135,114 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#Import CSV of Zipcodes\n",
+    "# Import CSV of ZipCodes\n",
     "d_zip = cudf.read_csv(\n",
     "    path_of(\"USA_Zipcodes_2019_Tiger.csv\"),\n",
     "    usecols=[\"WKT\", \"ZCTA5CE10\", \"INTPTLAT10\", \"INTPTLON10\"])\n",
     "d_zip.INTPTLAT10 = d_zip.INTPTLAT10.astype(\"float\")\n",
     "d_zip.INTPTLON10 = d_zip.INTPTLON10.astype(\"float\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "50b8d8bc-378f-4faa-b60c-e8f0ff507b2a",
+   "metadata": {},
+   "source": [
+    "The geometries are stored in [Well Known Text (WKT)](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) format.\n",
+    "Parsing the geoseries to geometry objects on host is possible, but can be very slow (uncomment to run):"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": 6,
    "id": "583856a2-0b29-48c1-ade7-3840f640c5e7",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Load WKT as shapely objects\n",
-    "h_zip = d_zip.to_pandas()\n",
-    "h_zip[\"WKT\"] = h_zip[\"WKT\"].apply(wkt.loads)\n",
-    "h_zip = gpd.GeoDataFrame(h_zip, geometry=\"WKT\", crs='epsg:4326')\n",
+    "# %%time\n",
+    "# # Load WKT as shapely objects\n",
+    "# h_zip = d_zip.to_pandas()\n",
+    "# h_zip[\"WKT\"] = h_zip[\"WKT\"].apply(wkt.loads)\n",
+    "# h_zip = gpd.GeoDataFrame(h_zip, geometry=\"WKT\", crs='epsg:4326')\n",
+    "\n",
+    "# # Transfer back to GPU with cuSpatial\n",
+    "# d_zip = cuspatial.from_geopandas(h_zip)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6fdaedfc-2d7f-4d73-a9b8-e3a8131bea2f",
+   "metadata": {},
+   "source": [
+    "Instead, we can use cudf list and string method to parse the wkt into coordinates and build a geoseries.\n",
+    "Without roundtripping to host, cudf provides ~40x speed up by computing on GPU. \n",
```

Comment:
This is awesome! e.g. Might need to state details of this comparison. e.g. which GPU and CPU.

Reply:
I remember this is mentioned below:

```
Reference machine: Intel(R) Xeon(R) CPU E5-2698 v4 @ 2.20GHz v.s. NVIDIA Tesla V100 SXM2 32GB
```